pkg/webhook/server.go

```diff
@@ -76,6 +79,10 @@ type Server struct {
 	// "", "1.0", "1.1", "1.2" and "1.3" only ("" is equivalent to "1.0" for backwards compatibility)
 	TLSMinVersion string
 
+	// TLSCiphers is used to specify the cipher algorithms that are negotiated
+	// during the TLS handshake, refer to https://pkg.go.dev/crypto/tls#CipherSuites
+	TLSCiphers []string
```

Comment:
Why not just make it `TLSOpts []func(*tls.Config)` to avoid needing a new setting everytime someone finds something new they would like to configure about TLS?

Reply:
Thanks, that makes sense!
I have kept the previously-existing TLS fields to not break existing usages